### PR TITLE
#30 - "Remember me" feature

### LINKED ISSUE #30: Make a feature to remember the user when logging in
Make the CheckBox "Запомнить меня" in the authorization.
image

### PR DESCRIPTION
**Pull request by ImmortalIdiot**
I created the "remember me" feature already in LoginFragment

### CODE REVIEW

**Review by NN171 — APPROVED** [review]

File: app/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -4,117 +4,134 @@
...
         android:layout_height="wrap_content"
-        android:gravity="center"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toBottomOf="@+id/flow"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/textInputLayout2"
         app:layout_constraintVertical_bias="0.191">
 
+        <androidx.appcompat.widget.AppCompatCheckBox
```
Comment (Xandr070) [inline app/src/main/res/layout/fragment_login.xml:93]: Need to change the color to main_blue

File: app/src/main/res/layout/fragment_login.xml [hunk 2]
```diff
@@ -4,117 +4,134 @@
...
     android:layout_height="match_parent"
-    android:orientation="vertical"
     android:gravity="center"
-    android:theme="@style/Theme.StudentApp"
-    android:padding="60dp"
+    android:orientation="vertical"
+    android:padding="16dp"
     tools:context=".auth.LoginFragment">
 
     <androidx.constraintlayout.helper.widget.Flow
```
Comment (Xandr070) [inline app/src/main/res/layout/fragment_login.xml:12]: We have abandoned the Flow container, but it is not critical, let it remain. If anything, I'll fix it after the merger

**Review by Xandr070 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_login.xml (hunk 1 above)
Comment (Xandr070) [inline app/src/main/res/layout/fragment_login.xml:93]: Need to change the color to main_blue in AppCompatCheckBox

File: app/src/main/res/layout/fragment_login.xml (hunk 2 above)
Comment (Xandr070) [inline app/src/main/res/layout/fragment_login.xml:12]: We have abandoned the flow container, but this is not critical, as a last resort, I will fix it myself

Comment (ImmortalIdiot) [reply app/src/main/res/layout/fragment_login.xml:12]: Ok, thank you

File: app/src/main/res/layout/fragment_login.xml (hunk 1 above)
Comment (ImmortalIdiot) [reply app/src/main/res/layout/fragment_login.xml:93]: Done

**Review by Xandr070 — APPROVED** [review]
It's really cool that you created a style for the checkbox

**Pull request merged**